src/AdminConsole/Components/Pages/App/Settings/AuthenticationConfiguration/EditAuthenticationConfiguration.razor

```diff
@@ -0,0 +1,46 @@
+@page "/app/{AppId}/settings/authentication-configuration/{Purpose}/edit"
+@using Passwordless.Common.Models.Apps
+@inherits BaseApplicationPage
+
+@inject IScopedPasswordlessClient Passwordless
+@inject ICurrentContext CurrentContext
+@inject NavigationManager NavigationManager
+@inject IHttpContextAccessor HttpContextAccessor
+
+<Panel>

```

Comment:
Still need to wrap it in a <Page> ... </Page> to set the title uniformly of the page.